bedrock/firefox/templates/firefox/family/includes/agreement.html

```diff
@@ -4,20 +4,36 @@
  file, You can obtain one at https://mozilla.org/MPL/2.0/.
 #}
 
+{% from "firefox/family/includes/macros.html" import module_tag %}
+
```

Comment:
**Question**: Noticed on smaller screens the module tag is centered, while in other sections it is left-aligned - and in the designs this section doesn't have a module tag in the mobile view. Not sure what to do about it, but just wanted to bring it up!

Reply:
yeah, this was a last minute update from design after seeing the non-styled Agreement heading on demo, I think it should definitely still be displayed on mobile, but can ask for feedback on what alignment they want, will merge as-is for now